paywall/src/utils/postOffice.js

```diff
@@ -0,0 +1,93 @@
+let handlers = {}
+
+/**
+ * postMessage manager
+ *
+ * This abstracts the setting up of listeners and senders for postMessage
+ * It handles all security-related aspects and allows setting handlers that
+ * respond to specific message types.
+ *
+ * A message is of format: { type: 'type', payload: <any> }
+ *
+ * Handlers are passed teh payload and a callback that can be used to send
```

Comment:
```suggestion
 * Handlers are passed the payload and a callback that can be used to send
```

Reply:
nice catch, clearly I can't read the font size/color on my machine...